### PR TITLE
Allow localized url

### LINKED ISSUE #458: Add support for different URL for different languages?
When a page is available in several languages, it is better to place corresponding versions' URL.

### PR DESCRIPTION
**Pull request by tupaschoal**
Closes #458 

Let me know what you think

### CODE REVIEW

File: script/validate_json.rb [hunk 1]
```diff
@@ -13,10 +13,17 @@ module ExitCodes
     MISSING_DOMAINS = 5        # Entry missing the required 'domains' field
     MISSING_NAME = 6           # Entry missing the required 'name' field
     UNEXPECTED_DIFFICULTY = 7  # Unexpected value for 'difficulty' field
+    UNEXPECTED_LANGUAGE = 8    # Unexpected language code for 'url_code' field
 end
 
 SupportedDifficulties = ["easy", "medium", "hard", "impossible"]
 
+def get_supported_languages()
+    return translation_files = Dir.glob('_data/trans/*').select { |f| File.file?(f) }.map { |f| f[12..-6] }
```
[the remark this answers is not in the document]
Comment (tupaschoal) [reply script/validate_json.rb:22]: Neat, I knew there would be something better, thanks :)

File: script/validate_json.rb [hunk 2]
```diff
@@ -39,6 +46,18 @@ def validate_difficulty(key)
     end
 end
 
+def validate_localized_urls(key)
+    key.keys.each do |entry_key|
+      if entry_key.include?("url_") && !SupportedLanguages.include?(entry_key)
```
[the remark this answers is not in the document]
Comment (tupaschoal) [reply script/validate_json.rb:51]: Nice catch, it was actually faulty testing, I tested for an invalid `url_something` but didn't for a valid one :P

**Comment by Nightfirecat** [comment]
LGTM, but I need to pull this down and give it a quick test before I properly approve. Also, this will need to be rebased once we merge #467

**Comment by Nightfirecat** [comment]
Oh, I would have thought the script splitting would have required it. I guess the changed sections weren't touched so it was fine. :shrug:

**Pull request merged**